OpenEdX/Info.plist

```diff
@@ -27,9 +29,16 @@
 	</dict>
 	<key>UIAppFonts</key>
 	<array/>
+	<key>UILaunchScreen</key>
+	<dict>
+		<key>UIColorName</key>
+		<string>SplashBackground</string>
+		<key>UIImageName</key>
+		<string>appLogo</string>
+		<key>UIImageRespectsSafeAreaInsets</key>
+		<true/>
+	</dict>
```

Comment:
Is there any chance not to replace the `LaunchScreen.storyboard` with the `Info.plist` configuration?
For some OeX instances we need a slightly more customizable LaunchScreen than just a background color and image, and this would be much easier with a storyboard file.

Reply:
Maybe the white-label script can just remove the `Launch screen interface file base name` key from the `Info.plist` and add `UILaunchScreen`?

Reply:
Replacing the whole LaunchScreen.storyboard also an option 👍

Reply:
@volodymyr-chekyrta Yes, we can try both approaches. At first glance, replacing the `LaunchScreen.storyboard` file looks easier. @saeedbashir could you please prepare and add this file into `edx-mobile-config` repo  and I will add whitelabel config and script changes?
Thank you

Reply:
Thanks!

Reply:
@volodymyr-chekyrta It's re-added.